Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -375,7 +375,7 @@ - (void)didMoveToSuperview {
 }
 
 - (void)reloadData {
-    [self.calendarView reloadData];
+    [self invalidateVisibleCells];
```

Comment:
I'm in two minds about this.

On one hand, it seems wrong to not call `reloadData` on the calendar because that's what the exposed method is called. Consumers might be surprised by this.

On the other hand, this achieves the same thing as `reloadData` so consumers would already know how to use it and wouldn't expect the method to be called something else.

Reply:
If it achieves the same thing, why does it fix the issue?